src/js/background.js

```diff
@@ -4,6 +4,8 @@ import {createDpopHeader} from '@inrupt/solid-client-authn-core';
 var id;
 var secret;
 var tokenUrl;
+var domainFilter;
```

Comment:
Please have a look at [this](https://codeburst.io/javascript-var-let-or-const-which-one-should-you-use-2fd521b050fa) regarding the use of `var`.

Reply:
Changed in 25cb8895125bd8e147dbcf163b18068fd1029de4